src/timer.rs

```diff
@@ -0,0 +1,70 @@
+trait Timer: Sized {
```

Comment:
The file should contain the copyright and license notice.

Reply:
Done. I copied it from the main.rs, the year is 2019, I hope it is fine.